src/evaluation/evaluate-score.service.ts

```diff
@@ -31,39 +38,50 @@ export class EvaluateScoreService {
         throw new NotFoundException('Submitted test details not found');
       }
 
-      const testId: string =
-        '09dd6726-55d0-4224-93fe-880f2cb5efef' || userTest?.testId;
+      const testId: string = userTest?.test_id;
       const testDetails: Test = await this.getTest(testId);
       const evaluatedAnswers: EvaluatedAnswer[] = [];
-      let rawScore = 0;
+      let totalMaxScore = 0;
+      let totalRawScore = 0;
 
-      userTest?.answers?.forEach((answer: Answer) => {
-        const currQuestion: Question = testDetails?.questions?.find(
-          (question) => question?.id === answer?.questionId,
-        );
+      userTest?.submitted_answers?.forEach(
+        (submittedAnswer: SubmittedAnswer) => {
+          const currQuestion: Question = testDetails?.questions?.find(
+            (question) => question?.id === submittedAnswer?.question_id,
+          );
 
-        if (!currQuestion) {
-          throw new NotFoundException('Question not found in DB');
-        }
-        const evaluatedAnswer: EvaluatedAnswer = {
-          questionId: answer?.questionId,
-          questionScore: this.getScore(currQuestion, answer),
-          selectedAnswer: this.getSelectedOption(currQuestion, answer),
-        };
-        rawScore += evaluatedAnswer.questionScore;
-        evaluatedAnswers.push(evaluatedAnswer);
-      });
+          if (!currQuestion) {
+            throw new NotFoundException('Question not found in DB');
```

Comment:
Isnt it 'Question not found in text in DB'

Reply:
No. It is same as previous message `Question not found in DB`
![image](https://github.com/compass-workshop/quiz-api/assets/86454870/ba248c80-3afc-4e0c-8818-7bad50421fe8)